pkg/api/deep_copy_generated.go

```diff
@@ -1306,15 +1306,6 @@ func DeepCopy_api_NodeAddress(in NodeAddress, out *NodeAddress, c *conversion.Cl
 }
 
 func DeepCopy_api_NodeAffinity(in NodeAffinity, out *NodeAffinity, c *conversion.Cloner) error {
-	if in.RequiredDuringSchedulingRequiredDuringExecution != nil {
-		in, out := in.RequiredDuringSchedulingRequiredDuringExecution, &out.RequiredDuringSchedulingRequiredDuringExecution
-		*out = new(NodeSelector)
-		if err := DeepCopy_api_NodeSelector(*in, *out, c); err != nil {
-			return err
-		}
-	} else {
-		out.RequiredDuringSchedulingRequiredDuringExecution = nil
-	}
```

Comment:
Did you delete this manually? I'm trying to debug some weirdness I'm noticing with ./hack/update-generated-deep-copies.sh